### DISCUSSION TITLE
MDAnalysis AtomGroup gets overwritten instead of concatenated

### DISCUSSION THREAD

**Discussion by aam-10** [MDAnalysis/mdanalysis#4630] [Q&A]
Hello.
I am trying to concat a group of residues (each group consists of the same residues but with different coordinates). Instead the atom group gets overwritten. Here is the code:

```
u = mda.Universe("2_molec.pdb") # main universe
og_sugar_chain = u.select_atoms("resname 3VA or resname 3LB or resname 0SA or resname ROH") # Create copies from this
protein = u.select_atoms("not (resname 3VA or resname 3LB or resname 0SA or resname ROH)")

sugar_chains = mda.AtomGroup([], u) # AtomGroup to contain the chains
residue = [34,56]
for residue in residues:
        sugar_chain = og_sugar_chain.copy()

        # Amino acid hydrogen atom
        for atom in protein.residues[residue].atoms:
            if atom.name == 'HG1':
                target_coords = atom.position
                break

        # Rotate
        r = R.from_euler('x', 270, degrees=True)
        r2 = R.from_euler('y', 90, degrees=True)

        for atom in sugar_chain.atoms:
            atom.position = r.apply(atom.position)
            atom.position = r2.apply(atom.position)

        # Translate
        base_sugar_atom_position = sugar_chain.residues[0].atoms[1].position

        translate_cords = [target_coords[0] - base_sugar_atom_position[0], target_coords[1] - base_sugar_atom_position[1], target_coords[2] - base_sugar_atom_position[2]]
    
        for atom in sugar_chain.atoms:
            move_coord = []
            for mol_cord, target_c in zip(atom.position, translate_cords):
                move_coord.append(mol_cord+target_c)
            atom.position = move_coord

        # Create atom group containing all oligosaccharide chains
        print(sugar_chains)
        sugar_chains = sugar_chains + sugar_chain # concat
        print(sugar_chains)
```

**Comment by richardjgowers** [comment]
It looks like you're trying to add atoms to a Universe using AtomGroup.copy. It isnt possible to add atoms to a Universe this way. I suspect what's happening is you are editing the position of the same group (sugar_chain) twice and losing some modifications.

**Reply by orbeckst** [reply]
To add: If you want to affect the current universe, use AtomGroup methods such as [AtomGroup.translate()](https://docs.mdanalysis.org/stable/documentation_pages/core/groups.html#MDAnalysis.core.groups.AtomGroup.translate) and [AtomGroup.rotate()](https://docs.mdanalysis.org/stable/documentation_pages/core/groups.html#MDAnalysis.core.groups.AtomGroup.rotate).

If you want to create a new Universe from existing AtomGroups, use [Merge()](https://docs.mdanalysis.org/stable/documentation_pages/core/universe.html#MDAnalysis.core.universe.Merge) and see also the User Guide on [Constructing, modifying, and adding to a Universe](https://userguide.mdanalysis.org/stable/examples/constructing_universe.html).
